src/components/common/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useSetTheme } from '../state/ThemeProvider';
+import Wrapper from './Wrapper';
+
+const Header = () => {
+  const { handleTheme } = useSetTheme();
+
+  const headerStyle = {
+    border: '1px solid black',
+  };
+
+  return (
+    <Wrapper>
+      <header style={headerStyle}>
+        <h1>Home Page</h1>
+        <div>
+          <label>
+            <input name="theme" type="radio" value="white" defaultChecked onChange={handleTheme} />
```

Comment:
radio buttons work! you could also just use a button and toggle it...or a checkbox! Lots of different ways  to do this.